commands/backups/schedule.js

```diff
@@ -0,0 +1,64 @@
+'use strict'
+
+const co = require('co')
+const cli = require('heroku-cli-util')
+
+const TZ = {
+  'PST': 'America/Los_Angeles',
+  'PDT': 'America/Los_Angeles',
+  'MST': 'America/Boise',
+  'MDT': 'America/Boise',
+  'CST': 'America/Chicago',
+  'CDT': 'America/Chicago',
+  'EST': 'America/New_York',
+  'EDT': 'America/New_York',
+  'Z': 'UTC',
+  'GMT': 'Europe/London',
+  'BST': 'Europe/London',
+  'CET': 'Europe/Paris',
+  'CEST': 'Europe/Paris'
+}
+
+function parse (at) {
+  let m = at.match(/^([0-2]?[0-9]):00 ?(\S*)$/)
+  if (!m) throw new Error("Invalid schedule format: expected --at '[HOUR]:00 [TIMEZONE]'")
+  let [, hour, timezone] = m
+
+  return {hour, timezone: TZ[timezone.toUpperCase()] || timezone || 'UTC'}
+}
+
+function * run (context, heroku) {
+  const host = require('../../lib/host')
+  const fetcher = require('../../lib/fetcher')(heroku)
+
+  const {app, args, flags} = context
+
+  let db = yield fetcher.addon(app, args.database)
+
+  let schedule = parse(flags.at)
```

Comment:
Might as well parse this before fetching the DB above--there's no point in continuing if the time is invalid.